app/templates/404.html

```diff
@@ -5,7 +5,7 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>You got 404'd - HTTP 404 - Page Not Found - Open Event</title>
+    <title>{{_('You got')}} 404'd - HTTP 404 - {{_('Page Not Found')}} - {{_('Open Event')}}</title>
```

Comment:
Yes, the name of the service is generic. Someone else could set it up using a different name for example.